cogs/adventofcode.py

```diff
@@ -0,0 +1,307 @@
+import asyncio
+import logging
+import datetime
+import aiohttp
+import pytz
+import re
+import inflect
+
+from bs4 import BeautifulSoup
+from datetime import datetime, timedelta
+from unicodedata import normalize
+
+import discord
+from discord.ext import commands
+
+
+log = logging.getLogger(__name__)
+loop = asyncio.get_event_loop()
+
+API_URL = 'https://adventofcode.com/2020/leaderboard/private/view/975452.json'
+AOC_CHANNEL = 778324114213175323
+INTERVAL = 120
+AOC_REQUEST_HEADER = {"user-agent": "TWT AoC Event Bot"}
+AOC_SESSION_COOKIE =  {'session': '.....'}
```

Comment:
move the session to the config file

Reply:
I put it in here, because it is used once a year, and not required by contributors during every setup